backend/dataall/base/cdkproxy/cdk_cli_wrapper.py

```diff
@@ -63,7 +63,7 @@ def update_stack_output(session, stack):
     aws = SessionHelper.remote_session(stack.accountid, stack.region)
     cfn = aws.resource('cloudformation', region_name=stack.region)
     try:
-        stack_outputs = cfn.Stack(f'{stack.name}').outputs
+        stack_outputs = cfn.StackRepository(f'{stack.name}').outputs
```

Comment:
This is not referring to the data.all Stack class but to the aws resource cfn.Stack!

Reply:
Oh, the naming was confusing...